src/seedu/addressbook/commands/EditNameCommand.java

```diff
@@ -0,0 +1,51 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
```

Comment:
Do not import everything using `*`; Import the necessary classes only, explicitly.